### PR TITLE
Changes made by request in last PR

### PR DESCRIPTION
**Pull request by Kyrevam**

### CODE REVIEW

File: src/main/java/pages/akaunting/AddNewCustomerPage.java [hunk 1]
```diff
@@ -21,18 +24,16 @@ public class AddNewCustomerPage extends BasePageInteractions {
     private By zipCodeField = By.id("zip_code");
     private By stateField = By.id("state");
     private By countryField = By.cssSelector("input[placeholder='- Select Country -']");
-
+    private By canLoginChk = By.cssSelector("label[for='create_user']");
+    //WebElement canLoginChk = eventDriver.findElement(By.id("create_user"));
```
Comment (martinhel2001) [inline src/main/java/pages/akaunting/AddNewCustomerPage.java:28]: Please remove commented lines like this.

File: src/test/java/UI_tests/AkauntingTests.java [hunk 2]
```diff
@@ -2,45 +2,40 @@
...
-
-
-
-
-
-
-
-        //Assert.assertTrue();
+        addNewCustomerPage.addNewCustomer(CustomerName, CustomerEmail);
+        // Assert.assertEquals(NewCustomerBanner.isDisplayed(), "Succesfull customer creation banner is not displayed");
```
Comment (martinhel2001) [inline src/test/java/UI_tests/AkauntingTests.java:36]: Remove commented lines.

File: src/test/java/UI_tests/AkauntingTests.java [hunk 3]
```diff
@@ -2,45 +2,40 @@
...
+        System.out.println("Texto de banner: "+eventDriver.findElement(NewCustomerBanner).getText());
```
Comment (martinhel2001) [inline src/test/java/UI_tests/AkauntingTests.java:37]: This printLn is not needed now.

**Review by martinhel2001 — CHANGES_REQUESTED** [review]
There are some changes needed before PR is approved.

File: src/main/java/pages/akaunting/AddNewCustomerPage.java (hunk 1 above)
Comment (Kyrevam) [reply src/main/java/pages/akaunting/AddNewCustomerPage.java:28]: Done!

File: src/test/java/UI_tests/AkauntingTests.java (hunk 3 above)
Comment (Kyrevam) [reply src/test/java/UI_tests/AkauntingTests.java:37]: Done!

**Review by martinhel2001 — APPROVED** [review]

**Pull request merged**